README.md

```diff
@@ -21,6 +21,7 @@ Extend `camunda-licensed/*` in your base eslint configuration:
 We ship with a few blue prints:
 
 * `camunda-licensed/mit`: ensures that all source files start with the [MIT license header](./resources/MIT-license-header.js)
+* `camunda-licensed/camunda`: ensures that all source files start with the Camunda commercial license
```

Comment:
Just a thought: Shouldn't we call the Camunda commercial license `commercial` instead of `camunda`? It would also reflect the fact that we have only one commercial license (as discussed previously).

Reply:
Can we link the license header here (similar to the MIT license entry)?

Reply:
* You're right, I will change it to `commercial`
* I will add the link.